case-lib/lib.sh

```diff
@@ -293,6 +293,24 @@ find_ldc_file()
     printf '%s' "$ldcFile"
 }
 
+func_mtrace_collect()
+{
+    local clogfile=$LOG_ROOT/mtrace.txt
+
+    if [ -z "$MTRACE" ]; then
+        MTRACE=$(command -v mtrace-reader.py) || {
+            dlogw 'No mtrace-reader.py found in PATH'
+            return 1
+        }
+    fi
+
+    local mtraceCmd="$MTRACE"
```

Comment:
In the last patch you use:
```bash
local mtraceBin; mtraceBin=mtrace-reader.py
```

Which one is the preferred one?

Reply:
Because in  `func_mtrace_collect()`, the `local mtraceCmd="$MTRACE"` can be set when testing with IPC4-Zephyr platforms.

So in `hijack.sh`, if we test on non-IPC4 Zephyr platforms. the mtraceCmd will be empty, we cannot reuse `mtraceCmd` in `hijack.sh`. So I set mtrace separately in these 2 scripts.